ReactiveCocoa/Swift/Property.swift

```diff
@@ -161,7 +161,9 @@ extension MutableProperty: SinkType {
 
 infix operator <~ {
 	associativity right
-	precedence 90
+
+	// Binds tighter than assignment but looser than everything else.
+	precedence 91
```

Comment:
Maybe 92 or 93? Changing precedence later is technically a breaking change.

Also, can you document that this is (intentionally) less than `|>`?